bolt-modules/boltlib/spec/functions/run_plan_spec.rb

```diff
@@ -78,6 +82,11 @@
         is_expected.to run.with_params('test::run_me_int', 'x' => 'should not work')
                           .and_raise_error(/expects an Integer value/)
       end
+
+      it 'fails with argument error if given nodes positional argument and nodes named argument' do
```

Comment:
It might also be worth adding a test that just `nodes =>` also works? I don't see any fixtures that use that...

Reply:
Also one that a plan that doesn't have nodes fails if nodes are specified (per Alex's comment)

Reply:
Added a test that calling `run_plan(name, targets)` on a plan without a `nodes` argument fails.

Re: testing that `run_plan(name, nodes => targets)` works, does that exercise anything that this existing test doesn't?

```
run_plan(name, hash) where hash is mapping argname to value
```

Easy enough to add if so.

Reply:
Ah I don't think so - I naively searched for `nodes =>` in the spec instead of a generic params passing test :) LGTM!